File: Unidad3/C#/u3p16.cs
using System;

class Program
{
    static void Main()
    {
        int c = 1, hrs, pago, sueldo;
        while (c <= 20)
        {
            Console.WriteLine("Captura las horas trabajadas");
            hrs = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Captura el pago por horas");
            pago = Convert.ToInt32(Console.ReadLine());

            sueldo = hrs * pago;
            Console.WriteLine("Tu sueldo es: " + sueldo);
            c++;
        }
    }
}

File: Unidad3/Java/U3p16.java
package com.mycompany.u3p16;
import java.util.Scanner;
public class U3p16 {

    public static void main(String[] args) {
        Scanner XD = new Scanner (System.in);
        int c=1, hrs, pago, sueldo;
        while (c<=20){
        System.out.println("Captura las horas trabajadas");
        hrs = XD.nextInt();
        System.out.println("Captura el pago por horas");
        pago = XD.nextInt();
        
        sueldo=hrs*pago;
        System.out.println("Tu sueldo es: "+sueldo);
        c++;
    }
    }
}
